src/List.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import * as R from 'ramda';
+import Item from './Item';
+
+export default function List({ items, onClick }) {
+  const element = (R.isEmpty(items))
+    ? <p>할 일이 없어요!</p>
+    : (
+      <ol>
+        {items.map((item) => (
+          <Item
+            content={item.content}
+            onClick={() => onClick(item.id)}
+          />
+        ))}
+      </ol>
+    );
```

Comment:
```suggestion
  const isEmpty = (array) => (R.isEmpty(array));
  
  if (isEmpty) {
    return <p>할 일이 없어요!</p>;
  }
  
  return (
    <ol>
      {items.map((item) => (
        <Item
          content={item.content}
          onClick={() => onClick(item.id)}
        />
      ))}
    </ol>
  );
```
위와 같이 `Guard Clauses`를 사용하면 할 일이 없는 경우에 대한 의도를 드러낼 수 있을 것 같아요~

참고:
- https://refactoring.com/catalog/replaceNestedConditionalWithGuardClauses.html

Reply:
`Guard Clauses`를 사용하는 것이 훨씬 읽기 좋네요. `a2813cb`에서 수정했습니다.

그런데 여기에서
```javascript
  const isEmpty = (array) => (R.isEmpty(array));
  
  if (isEmpty) {
    return <p>할 일이 없어요!</p>;
  }
```
위 처럼 하는 것이 다음보다 좋은 이유가 무엇인가요?

```javascript
  if (R.isEmpty(array)) {
    return <p>할 일이 없어요!</p>;
  }
```

Reply:
앗 이 부분은 람다를 제대로 파악하지 못한 상태에서 예시를 작성하다보니 저렇게 작성한것 같아요 😅

아래에 말씀하신대로 처리하는게 좋을 것 같습니다!